src/parsers/urdf.hpp

```diff
@@ -10,6 +10,17 @@
 #include "pinocchio/multibody/geometry.hpp"
 #include "pinocchio/parsers/urdf/types.hpp"
 
+/// \cond
+// Commented because this is needed in function prototypes.
+//#ifdef PINOCCHIO_WITH_HPP_FCL
+namespace fcl
+{
+  class MeshLoader;
+  typedef boost::shared_ptr<fcl::MeshLoader> MeshLoaderPtr_t;
```

Comment:
I would prefer `MeshLoaderPtr` instead of `MeshLoaderPtr_t`in order to make uniform the naming conventions in Pinocchio (still in progress yet).